c++/capnp.pc.in

```diff
@@ -1,11 +1,11 @@
 prefix=@prefix@
-exec_prefix=@exec_prefix@
```

Comment:
Ok, my question was stupid: 
The directory for object files and libraries of object code. Do not install executables here, they probably ought to go in $(libexecdir) instead. The value of libdir should normally be /usr/local/lib, but write it as $(exec_prefix)/lib. (If you are using Autoconf, write it as ‘@libdir@’.)


> Am 04.03.2016 um 00:58 schrieb Kenton Varda <notifications@github.com>:
> 
> In c++/capnp.pc.in:
> 
> > @@ -1,11 +1,11 @@
> >  prefix=@prefix@
> > -exec_prefix=@exec_prefix@
> I missed this when reviewing this before, but removing these lines breaks these files. #289 fixes it.
> 
> —
> Reply to this email directly or view it on GitHub.
>